docs/API.md

```diff
@@ -363,3 +430,153 @@ await realm.close()
 ```
 
 No output
+
+---
+
+#### getBackups
+
+() => Promise\<Backup[]>
+
+Gets a list of backups for the Realm
+
+```js
+await realm.getBackups()
+```
+
+<details>
+<summary>Output</summary>
+
+[Backup](#Backup)[]
+
+</details>
+
+---
+
+#### getWorldDownload
+
+() => Promise\<Download>
+
+Gets the most recent download for the Realm's world
+
+```js
+await realm.getWorldDownload()
+```
+
+<details>
+<summary>Output</summary>
+
+[Download](#Download)
+
+</details>
+
+---
+
+### Backup
+
+```js
+{
+    getDownload(): Promise<Download>
+    restore(): Promise<void>
+    id: string
+    lastModifiedDate: number
+    size: number
+    metadata: {
+        gameDifficulty: string
+        name: string
+        gameServerVersion: string
+        enabledPacks: string
```

Comment:
Docs for this field are outdated. You can either inline the fields into the Backup object or update the type here to be an object containing { resourcePack, behaviorPack }

Reply:
Fixed